examples/gallery/lines/hlines_vlines.py

```diff
@@ -0,0 +1,122 @@
+"""
+Horizontal and vertical lines
+=============================
+
+The :meth:`pygmt.Figure.hlines` and :meth:`pygmt.Figure.vlines`
+methods allow to plot horizontal and vertical lines in
+Cartesian, Geographic and Polar projections.
```

Comment:
Rewrap to 88 chars. 
```suggestion
The :meth:`pygmt.Figure.hlines` and :meth:`pygmt.Figure.vlines` methods allow to plot
horizontal and vertical lines in Cartesian, Geographic and Polar projections.
```
Hm. Acutally not sure about capitalizing "geographic" and "polar".
```suggestion
The :meth:`pygmt.Figure.hlines` and :meth:`pygmt.Figure.vlines` methods allow to plot
horizontal and vertical lines in Cartesian, geographic and polar projections.
```